cps/main.c

```diff
@@ -88,8 +88,7 @@ send_arp_reply_kni(struct cps_config *cps_conf, struct cps_arp_req *arp)
 	struct rte_mempool *mp;
 	int ret;
 
-	mp = cps_conf->net->gatekeeper_pktmbuf_pool[
-		rte_lcore_to_socket_id(cps_conf->lcore_id)];
+	mp = cps_conf->net->gatekeeper_pktmbuf_pool[cps_conf->lcore_id];
```

Comment:
In order to generalize the mbuf pools, let's have an inline function that receives `net` and `lcore_id` and returns the corresponding pool. This way, we are addressing the current change and another that may be needed.

Reply:
Eve better, given that each block will have is own pool, the pool should be part of the hidden fields of the configuration of the block or insurance of the block.